hack/generate-manifest.sh

```diff
@@ -31,6 +31,7 @@ Generate a YAML manifest for Antrea using Kustomize and print it to stdout.
         --np                          Generate a manifest with ClusterNetworkPolicy and Antrea NetworkPolicy features enabled
         --keep                        Debug flag which will preserve the generated kustomization.yml
         --tun (geneve|vxlan|gre|stt)  Choose encap tunnel type from geneve, gre, stt and vxlan (default is geneve)
+        --coverage                    Generate a manifest with measuring code coverage of Antrea binaries
```

Comment:
suggestion: Generates a manifest which supports measuring code coverage of Antrea binaries

Reply:
sounds good. will do that.

Reply:
Hi Antoni, I checked the code, there are multiple places that we use hack/generate-manifest.sh to generate antrea yaml files. For example, in ci/kind/test-e2e-kind.sh.
$YML_CMD --kind --encap-mode $current_mode $manifest_args

The --coverage param needs to work with other params like --np --proxy etc to generate the final yaml files.
If I do not thought it the wrong way, it will have many duplicate code in generate-manifest-coverage.sh. 
So I added --coverage as a optional param here. 

Would like to know your comments.

Reply:
please update the command-line option as suggested: "Generates a manifest which supports measuring code coverage of Antrea binaries."

otherwise changes in this file look good to me

Reply:
Done